twine/__main__.py

```diff
@@ -15,26 +15,19 @@
 import sys
 from typing import Any
 
+import colorama
 import requests
 
 from twine import cli
 from twine import exceptions
 
-try:
-    import colorama
-except Exception:
-    colorama = None
-
 
 def main() -> Any:
     try:
-        if colorama:
-            colorama.init()
+        colorama.init()
```

Comment:
Can `colorama.init()` raise an exception? If so, this try/except block almost certainly won't handle it. Either move this to its own `try/except` or move it before the `try/except` block. This doesn't need to be here and it confuses future readers as to what is raising the TwineException and requests.HTTPError exceptions

Reply:
Good point, looking at [colorama/initialise.py](https://github.com/tartley/colorama/blob/master/colorama/initialise.py), it seems that the only reasonable exception `colorama.init()` would raise is a ValueError dependent on a bad combination of arguments, so I'm just moving it out of the `try/except` for now.